### ISSUE TITLE
Cannot install 0.4.0-beta.0 with yarn v2

### ISSUE THREAD

**Issue by bugzpodder** [pmmmwh/react-refresh-webpack-plugin#95]
Not sure what happened between the builds, but the next version stopped working with yarn v2 (berry).

`yarn policies set-version berry` to change yarn version

It seem to be a new post_install command in the build?

The log says: 
```
# This file contains the result of Yarn building a package (@pmmmwh/react-refresh-webpack-plugin@virtual:c5162dee2ae0251137e7d2b232656b8153e2afb57deb8fe296e597abc4aeac8b0cfb0e63c158b4916cd3b8d4e10364fe6e238a3b8520d68c8bae0bf01d283571#npm:0.4.0-beta.0)
# Script name: postinstall

Unknown Syntax Error: Not enough positional arguments.

$ yarn link [--all] [-p,--private] [-r,--relative] <destination>
```

```
➤ YN0007: │ @pmmmwh/react-refresh-webpack-plugin@npm:0.4.0-beta.0 [c5162] must be built because it never did before or the last one failed
➤ YN0009: │ @pmmmwh/react-refresh-webpack-plugin@npm:0.4.0-beta.0 [c5162] couldn't be built successfully (exit code 1, logs can be found here: /private/var/folders/ss/hkytw8gj38v38b8tp2q_dvf80000gp/T/xfs-d65887fb/build.log)
➤ YN0009: │ @pmmmwh/react-refresh-webpack-plugin@npm:0.4.0-beta.0 [c5162] couldn't be built successfully (exit code 1, logs can be found here: /private/var/folders/ss/hkytw8gj38v38b8tp2q_dvf80000gp/T/xfs-d65887fb/build.log)
➤ YN0000: └ Completed in 1.46s
➤ YN0000: Failed with errors in 1.61s

 % yarn add -D @pmmmwh/react-refresh-webpack-plugin@0.3.1       
➤ YN0000: ┌ Resolution step
➤ YN0002: │ refresh@workspace:. doesn't provide react-refresh@^0.8.2 requested by @pmmmwh/react-refresh-webpack-plugin@npm:0.3.1
➤ YN0000: └ Completed in 0.4s
➤ YN0000: ┌ Fetch step
➤ YN0013: │ @pmmmwh/react-refresh-webpack-plugin@npm:0.3.1 can't be found in the cache and will be fetched from the remote registry
➤ YN0019: │ @pmmmwh-react-refresh-webpack-plugin-npm-0.4.0-beta.0-4ea71b1de4-3.zip appears to be unused - removing
➤ YN0000: └ Completed in 0.46s
➤ YN0000: ┌ Link step
➤ YN0000: └ Completed in 0.3s
➤ YN0000: Done with warnings in 1.19s
```

**Comment by bugzpodder** [comment]
Got it to work with   the following entry in package.json:
```
"dependenciesMeta": {
    "@pmmmwh/react-refresh-webpack-plugin": {
      "built": false
    }
  },
```
seems to work now.

**Comment by pmmmwh** [comment]
Thanks for reporting! #97 will fix this - once tests pass I will release `0.4.0-beta.1`.
